api/src/main/java/au/org/aodn/nrmn/restapi/util/ValidatorHelpers.java

```diff
@@ -22,14 +22,16 @@
                 .map(value -> invalid)
                 .orElse(valid);
     }
-  public <E,T> List<E> toErrorList(Validated<E, T>  validator) {
+
+    public static <E,T> List<E> toErrorList(Validated<E, T>  validator) {
         return  validator.bimap(Seq::of, Function.identity()).foldInvalidLeft(Monoids.seqConcat()).toList();
     }
 
 
-    public List<ErrorInput> reducetoErrrors(Validated<ErrorInput, String>  ...validators){
+    public static List<ErrorInput> reducetoErrrors(Validated<ErrorInput, String>  ...validators){
```

Comment:
if you change this method it would be good to Generify ErrorInput to E

Reply:
Sure, no problem.

Reply:
Done